modules/tile-converter/src/i3s-converter/helpers/feature-attributes.ts

```diff
@@ -102,6 +103,27 @@ export function getAttributeType(key: string, attribute: unknown): string {
   return STRING_TYPE;
 }
 
+/**
+ * Get the attribute type for attributeStorageInfo according to the schema
+ * @see https://github.com/Esri/i3s-spec/blob/master/docs/1.7/attributeStorageInfo.cmn.md
+ * @param {SchemaClassProperty} schema - attribute's schema
+ * @returns attribute's type
+ */
+export function getAttributeTypeBasedOnSchema(schema: SchemaClassProperty): string {
```

Comment:
Add a test for the new function

Reply:
Added tests for getAttributePropertiesFromSchema and createStorageAttributes.